notifications/models.py

```diff
@@ -1,10 +0,0 @@
-from django.db import models
-
-from patrons.models import Patron
-
-
-# Create your models here.
-class Notification(models.Model):
```

Comment:
need to add class Meta
add relevant helping + error messages
add __str__ funcation for the admin